### ISSUE TITLE
RC Webinar not syncing to HubSpot

### ISSUE THREAD

**Issue by jrheidkamp** [ringcentral/hubspot-embeddable-ringcentral-phone#224]
Listed on HS Support Page: https://support.ringcentral.com/s/article/9364?language=en_US

If you use Zoom + HubSpot, this data integrates and is pushed into HubSpot.    Can we get this feature added?

**Comment by zxdong262** [comment]
Do you mean manualy sync meeting to contacts? Check this: https://youtu.be/ajbipkPd_4Q?t=383

**Comment by jrheidkamp** [comment]
No.  Zoom & HubSpot integration allows for recording and tracking of detailed attendance at Webinars:

https://knowledge.hubspot.com/integrations/use-hubspots-integration-with-zoom

Looking to add this.

**Comment by grokify** [comment]
We are working on this and it may be available later this year.
